src/components/productGallery/ProductGallery.css

```diff
@@ -0,0 +1,87 @@
+.product-gallery-main {
+  height: 350px;
+  width: 100%;
+  display: flex;
+  gap: 7px;
+  margin-top: 30px;
+  margin-bottom: 50px;
+}
+.big-img-container {
+  width: 50%;
+  height: 100%;
+}
+.big-image {
+  cursor: pointer;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+}
+.big-image:hover {
+  opacity: 0.9;
+}
+.small-imgs-container {
+  width: 50%;
+  height: 100%;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 8px;
+  box-sizing: border-box;
+}
+.small-img-left {
+  cursor: pointer;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+.small-img-right-top {
+  cursor: pointer;
+  width: 100%;
+  height: 100%;
+  border-top-right-radius: 10px;
+  overflow: hidden;
+}
+.small-img-right-bottom {
+  cursor: pointer;
+  width: 100%;
+  height: 100%;
+  border-bottom-right-radius: 10px;
+  object-fit: fill;
+  overflow: hidden;
+}
```

Comment:
We can simplify it to:
.small-img-left, .small-img-right-top, .small-img-right-bottom {
cursor: pointer;
width: 100%;
height: 100%;
object-fit: cover;
overflow: hidden;
}
.small-img-right-top {
border-top-right-radius: 10px;
}
.small-img-right-bottom {
border-bottom-right-radius: 10px;
}